src/framework/audio/internal/plugins/knownaudiopluginsregister.h

```diff
@@ -25,15 +25,18 @@
 
 #include "audio/iknownaudiopluginsregister.h"
```

Comment:
I would suggest moving "audio/iknownaudiopluginsregister.h" to the very bottom and without "audio"

Reply:
fixed